src/screens/Podcast/Podcast.tsx

```diff
@@ -1,117 +1,121 @@
-      name: 'This is Berk',
-      createdAt: 'August 26, 2021'
-    }
-  ]
-}
+  useEffect(() => {
+    dispatch(loadPodcastAction(Number(id)))
+    dispatch(
+      loadEpisodesByPodcastIdAction({
+        podcastId: Number(id),
+        filter: DEFAULT_EPISODES_PAGINATION
+      })
+    )
+  }, [])
 
-type Props = {
-  podcast: Podcast
-}
-
-const PodcastPage: React.FC<Props> = ({ podcast }) => {
   return (
-    <ScrollView>
-      <View>
-        <ImageBackground
-          source={{ uri: mockedPodacst.background }}
-          resizeMode="cover"
-          style={styles.podcastBackground}
-        >
-          <TouchableOpacity style={styles.backButton} activeOpacity={0.7}>
-            <BackButton width={40} />
-          </TouchableOpacity>
-          <View style={styles.podcastLogoContainer}>
-            <Image
-              source={{ uri: mockedPodacst.image }}
+    <ScrollView contentContainerStyle={styles.scrollView}>
+      {podcast ? (
+        <>
+          <View>
+            <ImageBackground
+              source={{ uri: podcast.cover?.url ?? DEFAULT_IMAGE }}
               resizeMode="cover"
-              style={styles.podcastLogo}
+              style={styles.podcastBackground}
+            >
+              <TouchableOpacity style={styles.backButton} activeOpacity={0.7}>
+                <BackButton width={40} />
+              </TouchableOpacity>
+              <View style={styles.podcastLogoContainer}>
+                <Image
+                  source={{ uri: podcast.image?.url ?? DEFAULT_IMAGE }}
+                  resizeMode="cover"
+                  style={styles.podcastLogo}
+                />
+              </View>
+            </ImageBackground>
+          </View>
+          <View style={styles.podcastNameContainer}>
+            <Heading
+              label={podcast.name}
+              type={HeadingType.LARGE}
+              style={styles.podcastName}
+            />
+          </View>
+          <View style={styles.podcasterNameContainer}>
+            <PlainText
+              label={podcast.user.nickname}
+              style={styles.podcastAuthorText}
+            />
+          </View>
+          <View style={styles.description}>
+            <PlainText
+              label={podcast.description}
+              style={styles.descriptionText}
+            />
+          </View>
+          <View style={styles.episodeCounter}>
+            <CircleIcon width={5} />
+            <PlainText
+              label={`${episodes.length} Episodes`}
+              style={styles.edisodesCount}
+            />
+          </View>
+          <View style={styles.episodesContainer}>
+            <Heading
+              type={HeadingType.MEDIUM}
+              label={`Episodes`}
+              style={styles.episodesContainerTitle}
             />
+            {hasEpisodes ? (
+              episodes.map((episode: Episode, inx: number) => (
+                <EpisodeItem episode={episode} number={inx} key={episode.id} />
```

Comment:
I think it will be better if calculate `position` number here. Logic of our `EpisodeItem` should be independent on this detail.
```suggestion
                <EpisodeItem episode={episode} number={idx + INCREASE_VALUE_FOR_LIST} key={episode.id} />
```